py/desispec/io/photo.py

```diff
@@ -0,0 +1,862 @@
+"""
+desispec.io.photo
+=================
+
+Simple methods for gathering photometric and targeting catalogs.
+
+"""
+import os, pdb
+from glob import glob
+import numpy as np
+import fitsio
+
+from astropy.table import Table, vstack
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+from desitarget.io import desitarget_resolve_dec
+
+from desiutil.log import get_logger, DEBUG
+log = get_logger()#DEBUG)
+
+def gather_targetdirs(tileid, fiberassign_dir=None):
+    """Gather all the targeting directories used to build a given fiberassign catalog.
+
+    Args:
+        tileid (int): tile number
+        fiberassign_dir (str, optional): directory to fiberassign tables
+
+    Returns a list of all the unique targeting directories.
```

Comment:
If there are Target of Opportunity (TOO) targets on the tile, this also returns ToO *file(s)* and not just directories, correct?  e.g. `photo.gather_targetdirs(100)` includes a file in the return.  I think that is expected behavior, but please document.

Reply:
Done in 82dab6d95243689c2bce89d11cdcbf89b10dcd7f.